canvas_manage_course/settings/isites_migration.py

```diff
@@ -1,17 +1,32 @@
+"""
+These settings are used for:
+- isites_migration.management.commands.export_slide_tool (just exporting slide
+    tool data; uses e.g.
+- jobs put on the RQ isites_export queue (which run
+    isites_migration.jobs.migrate_files)
+"""
 from .aws import *
 
 INSTALLED_APPS = (
-    'icommons_common',
     'async',
     'django_rq',
+    'icommons_common',
     'isites_migration',
+    'lti_school_permissions',
```

Comment:
IIRC suddenly you couldn't run the commands with these settings anymore using manage.py. I think it was something to do with parsing URL includes (the app name was not in the installed apps and it failed?). I can't remember the details, but happy to troubleshoot if needed.